Neos.ContentRepository.Export/src/Event/ValueObject/ExportedEvent.php

```diff
@@ -80,12 +80,8 @@ public function toJson(): string
         }
     }
 
-    /**
-     * @return array{identifier: string, type: string, payload: array<mixed>, metadata?: ?array<mixed>, streamName?: string, version?: int, sequenceNumber?: int, recordedAt?: string}
-     */
-    public function jsonSerialize(): array
+    public function jsonSerialize(): mixed
```

Comment:
This returns an array, we should not make the API less specific IMO, but I'd be fine with:

```suggestion
    /**
      *@return array<string, mixed>
      */
    public function jsonSerialize(): array
```

Reply:
ok, good points! the thing is, that we keep discussing these things and changing them back and forth.
IMO this change is not related to this PR

Reply:
okay yes that makes sense i will try to create a canvas where we can discuss new code styles and put agreed upon practices in a separate spot?

I changed the code as i wanted to get rid of the phpstan ignore, originally this change is of course picked together a bit and contains / solves separate concerns but i didnt want to just trash my local stash, thus i created a pr which kinda makes sense hopefully: Event (export) related cleanup